services/web-app/pages/pages.module.scss

```diff
@@ -5,10 +5,6 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-.content {
-  margin: $spacing-07 0;
-}
```

Comment:
@mattrosno I did a visual test to see if removing this would affect any styles. Let me know if I'm missing something and we need to keep this

Reply:
I'd search to see where that classname is all being used. I'm seeing it being used in quite a few pages.
![image](https://user-images.githubusercontent.com/1691245/149230907-d7c5ed01-4413-4de2-a465-5705a7c07d1b.png)